scss/bootstrap/bootstrap.scss

```diff
@@ -1,19 +1,17 @@
-@import "vendor/text-angular/text-angular";
-@import "vendor/angular-xeditable/angular-xeditable";
-@import "vendor/font-awesome/font-awesome";
-@import "vendor/perfect-scrollbar/main";
-@import "vendor/ui-bootstrap/ui-bootstrap";
-@import "vendor/ui-select/ui-select";
-@import "vendor/select2/select2-bootstrap";
+@import '../angular/vendor/*';
+@import 'vendor/font-awesome/font-awesome';
+@import 'vendor/perfect-scrollbar/main';
+@import 'vendor/select2/select2-bootstrap';
 
-@import "overrides/variables";
-@import "fonts/*";
-@import "mixins/*";
+@import 'overrides/variables';
+@import 'fonts/*';
+@import 'mixins/*';
 
-@import "SCSSROOT/org.civicrm.shoreditch/base/scss/bootstrap";
-@import "overrides/style/*";
-@import "overrides/vendors/*";
-@import "overrides/crm/*";
-@import "variants/*";
-@import "components/*";
-@import "helper";
+@import 'SCSSROOT/org.civicrm.shoreditch/base/scss/bootstrap';
+@import '../angular/overrides/*';
```

Comment:
why can't we have a single
```scss
@import 'angular/*';
```
?

Reply:
If we do `/*`, we cant choose which folder should be loaded first. And hence vendor gets loaded last and overrides the overridden styles.